test/Garnet.test/RespTests.cs

```diff
@@ -187,6 +187,10 @@ public void SetExpiryHighPrecision()
             string retValue = db.StringGet("mykey");
             Assert.AreEqual(origValue, retValue);
 
+            Thread.Sleep(1000);
```

Comment:
This test might fail CI periodically, because 0.1 secs is within the margin of CI random delays and swaps. Can you  make the original expiry more like 1.9 and sleep for 1 sec?

Reply:
Thanks for the careful review.